src/test/java/hwjdi/JdiTest.java

```diff
@@ -0,0 +1,58 @@
+package hwjdi;
+
+import com.epam.jdi.light.driver.WebDriverUtils;
+import hwjdi.entities.User;
+import hwjdi.beans.MetalColorEntity;
+import org.testng.Assert;
+import org.testng.annotations.AfterSuite;
+import org.testng.annotations.BeforeSuite;
+import org.testng.annotations.Test;
+
+import java.util.Map;
+
+import static com.epam.jdi.light.elements.init.PageFactory.initElements;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class JdiTest {
+
+    @BeforeSuite
+    public void setUp() {
+        initElements(Site.class);
+    }
+
+    @AfterSuite
+    public void tearDown() {
+        WebDriverUtils.killAllSeleniumDrivers();
+    }
+
+
+    @Test
+    public void metalAndColorSubmitFormTest() {
+        User roman = new User(DataProviders.login, DataProviders.password, DataProviders.userName);
```

Comment:
Basically, you should not create an entity in Test method. This functionality should be encapsulated somewhere else. At least, you can create a constant like we did in class work.

Reply:
fixed